### DISCUSSION TITLE
Please help, the reset password link wont work

### DISCUSSION THREAD

**Discussion by ehsan0921** [mastodon/mastodon#21376] [Q&A]
when users lost their password and then click on forget the password, they get the Email with a link.

but when users click on the confirmation email for the rest password they get this error.

You are logged in via an external service, so password and e-mail settings are not available.

why is that?
btw I'm using cloud on for the installation of Mastodon.

**Comment by mjankowski** [comment]
This happens when you have an external (PAM, LDAP, etc) auth system configured, and so the users cant reset their credentials within the app itself.
